src/webhook-mappers/signature-validator.ts

```diff
@@ -0,0 +1,37 @@
+import { createHmac } from "crypto";
+
+type SignatureType = 'sha1' | 'sha256';
+
+const createSignature = (signatureType: SignatureType, payload: string) => {
+    const hmac = createHmac(signatureType, 'a secret');
```

Comment:
Important: the secret is currently hard-coded. I.e you have to put "a secret" into the webhook configuration in GitHub. Later, this needs to be configured by the user themselves and stored in Forge's secure storage (I'm waiting to get early access to secure storage, as it's not generally available, yet)

Reply:
Gotta love when you're one of the 'Guinea pig users' of a new app.